src/components/super-networked-interactable.js

```diff
@@ -46,7 +46,7 @@ AFRAME.registerComponent("super-networked-interactable", {
     this.hand = e.detail.hand;
     if (this.networkedEl && !NAF.utils.isMine(this.networkedEl)) {
       if (NAF.utils.takeOwnership(this.networkedEl)) {
-        this.el.setAttribute("body", { mass: this.data.mass });
+        this.el.setAttribute("body", { type: "dynamic", mass: this.data.mass });
```

Comment:
Since we are actually toggling the type now we should be able to get rid of `mass` from this component's schema and just never touch the body's mass.

Reply:
toggling the type doesn't change the need for changing mass. Changing the mass is still required to get the behavior of switching between dynamic and static. Changing the type simply avoids unnecessary methods from being called on non-owned objects.